src/server/pkg/storage/chunk/storage.go

```diff
@@ -64,15 +69,42 @@ func (s *Storage) CreateSemanticReference(ctx context.Context, name string, chun
 	return s.gcClient.CreateReference(ctx, semanticReference(name, chunk.Hash))
 }
 
+// CreateSemanticReference creates a semantic reference to a chunk.
+func (s *Storage) CreateTemporaryReference(ctx context.Context, name string, chunk *Chunk, ttl time.Duration) (*time.Time, error) {
```

Comment:
Why are we passing around pointers to a `time.Time`? The Go recommendation is to pass by value for this type: https://golang.org/pkg/time/#Time